api/bash/gitline

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+DOMAIN="https://api.github.com"
+source ~/projects/github_bash_project/.privateVariables.sh
+
+# Assigning Arguments
+whichMethod=$1
+username="$2"
+label_state_or_repo_name=$3
+is_private=$4
+
+function fetchIssuesWhereIAmAssignedWithLabel {
+  response=$(curl --silent ${DOMAIN}/search/issues?q=+label:${label_state_or_repo_name}+assignee:${username}+state:open&sort=created&order=asc)
```

Comment:
Just chatted with @PatrickBM, GH_TOKEN is also what's in the other scripts in this repo as well too